### PR TITLE
[MSI] Fix empty disc costing

### PR DESCRIPTION
**Pull request by atharva1910**
Continuing from : [PR](https://github.com/reactos/reactos/pull/4966)
JIRA issue: [CORE-18749](https://jira.reactos.org/browse/CORE-18749)

Test
image

### CODE REVIEW

File: dll/win32/msi/dialog.c [hunk 1]
```diff
@@ -3157,8 +3157,17 @@ static void msi_dialog_vcl_add_columns( msi_dialog *dialog, msi_control *control
 
         /* the width must be a positive number
          * if a width is invalid, all remaining columns are hidden
+         * Skip in case of prefix the string of displayed characters with {\style} or {&style}.
```
Comment (JoachimHenze) [inline dll/win32/msi/dialog.c:3160]: Do not change the code outside of the ifdef block. Either undo it, or move that comment into the ifdef-block

File: dll/win32/msi/dialog.c [hunk 2]
```diff
@@ -3157,8 +3157,17 @@ static void msi_dialog_vcl_add_columns( msi_dialog *dialog, msi_control *control
...
          */
         if ( !wcsncmp( num, L"-", 1 ) || !str_is_number( num ) ) {
+#ifdef __REACTOS__
+            if (count == 0 && (!wcsncmp( num, L"\\", 1 ) || !wcsncmp( num, L"&", 1 )))
```
Comment (JoachimHenze) [inline dll/win32/msi/dialog.c:3164]: no whiteaspace padding at innerside of braces. Use
`if (count == 0 && (!wcsncmp(num, L"\\", 1) || !wcsncmp(num, L"&", 1)))`

File: dll/win32/msi/dialog.c [hunk 3]
```diff
@@ -3157,8 +3157,17 @@ static void msi_dialog_vcl_add_columns( msi_dialog *dialog, msi_control *control
...
+            {
+                FIXME("Style prefix not supported\n");
+                msi_free( num );
```
Comment (JoachimHenze) [inline dll/win32/msi/dialog.c:3167]: I would also omit the whitespace padding here. And yes, I know that it was copy-pasted from Wine below.

**Review by JoachimHenze — CHANGES_REQUESTED** [review]

**Comment by HBelusca** [comment]
What's the purpose of this present PR? Does it duplicate PR #4966 ? What does it do more? Does it aim at implementing the not-yet implemented suggestions from that other PR?

**Review by KRosUser2 — APPROVED** [review]
lgtm

File: dll/win32/msi/dialog.c (hunk 1 above)
Comment (atharva1910) [reply dll/win32/msi/dialog.c:3161]: Fixed

File: dll/win32/msi/dialog.c (hunk 3 above)
Comment (atharva1910) [reply dll/win32/msi/dialog.c:3173]: Its inside a if block, the padding is okay

Comment (HBelusca) [reply dll/win32/msi/dialog.c:3173]: the referred padding was those extra spaces after and before the opening and closing parentheses in the function calls.

File: dll/win32/msi/dialog.c (hunk 1 above)
Comment (JoachimHenze) [reply dll/win32/msi/dialog.c:3161]: No need for the #else block.
Just move that single comment line into your 2nd #ifdef block and you are done. You don't have to extend the existing comment.

Comment (atharva1910) [reply dll/win32/msi/dialog.c]: Okay, done

**Review by JoachimHenze — APPROVED** [review]

**Pull request merged**